src/Items/ItemHandler.cpp

```diff
@@ -411,6 +425,27 @@ bool cItemHandler::OnPlayerPlace(
 
 
 
+bool cItemHandler::GetBlocksToPlace(
+	cWorld & a_World, cPlayer & a_Player, const cItem & a_EquippedItem,
+	int a_BlockX, int a_BlockY, int a_BlockZ, eBlockFace a_BlockFace,
+	int a_CursorX, int a_CursorY, int a_CursorZ,
+	sSetBlockVector & a_BlocksToSet
+)
+{
+	BLOCKTYPE BlockType;
+	NIBBLETYPE BlockMeta;
+	if (!GetPlacementBlockTypeMeta(&a_World, &a_Player, a_BlockX, a_BlockY, a_BlockZ, a_BlockFace, a_CursorX, a_CursorY, a_CursorZ, BlockType, BlockMeta))
```

Comment:
I notice that this isn't called in places where its overridden. Maybe it should be placed somewhere which makes sure that it is, like a non-overridable caller that calls all of the virtual functions?